### PR TITLE
Test against Ruby 2.3.0 on Travis CI

### PR DESCRIPTION
**Pull request by koic**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,9 +2,11 @@ before_script:
  - git fetch --unshallow # we need a full clone for tests
  - git config --global user.email "you@example.com"
  - git config --global user.name "Your Name"
+ - gem install bundler
```
Comment (grosser) [inline .travis.yml]: fails without ?

Comment (koic) [inline .travis.yml]: Yes. https://github.com/travis-ci/travis-ci/issues/5239

But tests failed. (Ummm)
I will try to solve this problem. Please wait a moment.

Comment (koic) [inline .travis.yml]: I have solved the problem.
(Sorry, I made a mistake `before_install` for `before_script`.)

**Comment by grosser** [comment]
thx! :D

**Comment by koic** [comment]
Thank you for your merging.

**Pull request merged**